### DISCUSSION TITLE
Crash and error when I click plugin general options

### DISCUSSION THREAD

**Discussion by PerfectMoments** [rsjaffe/MIDI2LR#519] [Problems]
**Describe the bug**
When try to open the general options tab it restarts and gives me a "an  internal error has occured: [string "limits.lua"]:38: attempt  to index global 'ProgramPreferences' (a nil value)

**To Reproduce**
1. Go to "file"
2. Click on "plug-in extras"
3. Click on "general options"

**Expected behavior**
I do not know why and how it started to appear as I already set up my midi controller but wanted to do some changes... Now I am not able to personalized it in any other way

**Screenshots**
yes
![1](https://user-images.githubusercontent.com/90153439/132137856-710080b1-e9d8-4f7b-a344-d5f2a3315014.PNG)

**System(please complete the following information):**
 - OS: Windows 11
 - Lightroom classic ver. 10
 - MIDI2LR version 4.3.1.11

**Additional context**
Nope.

**Comment by rsjaffe** [comment]
Get out of the develop module, then open the dialog. What type of photo file is that (dng, jpg, etc.)?

**Comment by PerfectMoments** [comment]
> Get out of the develop module, then open the dialog. What type of photo file is that (dng, jpg, etc.)?

Will try that ASAP
Sony RAW files - .arw


P.S. - changing the work module does not fix it..

**Comment by misiaeeel** [comment]
I have that same issue, It's impossible to open that same error
GENERAL OPTIONS
KEYWORDS
DEVELOP PRESET

**Comment by rsjaffe** [comment]
Try the following. 

1. Upgrade to the latest version MIDI2LR https://github.com/rsjaffe/MIDI2LR/releases/latest.
2. Then, move the .lua files from [PrefReset.zip](https://github.com/rsjaffe/MIDI2LR/files/7453807/PrefReset.zip) into the MIDI2LR.lrplugin directory.
3. Then, start Lightroom. Open Lightroom's File → Plug-in Extras menu, select, under MIDI2LR, `Reset Settings`.
4. Close Lightroom (don't use MIDI2LR at this point, it may crash).
5. Restart Lightroom.

That should clear out any old errors in the settings. Did that help?

**Comment by PerfectMoments** [comment]
Hello,
Today I needed to reinstall windows, reinstall and update LightRoom and then reinstall midi2lr and saw that there is a newer version too....
That fixed the problem. But Thank you very much for your support!
